modules/test/playwright/pages/object-web/ModelBuilderPage.ts

```diff
@@ -218,6 +206,30 @@ export class ModelBuilderPage {
 		);
 	}
 
+	async openObjectFieldSelectionPage(
+		objectDefinitionName: string,
+		objectFieldBusinessType: string,
+		objectFieldLabel: string
+	) {
+		await this.leftSidebarItems
+			.filter({hasText: objectDefinitionName})
+			.click();
+
+		await this.objectDefinitionNodes
+			.filter({hasText: objectDefinitionName})
+			.getByRole('button', {name: 'Add Field or Relationship'})
+			.click();
+
+		await this.addObjectFieldButton.click();
```

Comment:
This function should end at this line, if you think about its name, it should only open the modal.